src/Build.UnitTests/BackEnd/RedirectConsoleWriter_Tests.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Build.Execution;
+using Xunit;
+
+namespace Microsoft.Build.Engine.UnitTests.BackEnd
+{
+    public class RedirectConsoleWriter_Tests
+    {
+        [Fact]
+        public async Task EmitConsoleMessages()
```

Comment:
minor: consider the Async suffix for async methods
https://docs.microsoft.com/en-us/dotnet/csharp/async

Reply:
Reason for Async suffix is to make user of that API get realize that this method shall be somehow awaited.
Since this is unit test and no user will eve call it directly, Async suffix would just be be a noice.